### ISSUE TITLE
Fetch latest commit and metadata for a course from GitHub when clicking refresh button in top navbar 

### ISSUE THREAD

**Issue by c21rebja** [HGustavs/LenaSYS#13668]
When pressing the refresh button in the navbar in a course, send the course ID to getLatestCommit.php and compare the latest commit with the commit saved in the SQLite database. 
If the course needs to be updated, fetch the metadata using FetchGithubRepo.php.
This also includes removing the function call to FetchGithubRepo.php from "create course". 
Also connect the functions in file getLatestCommit.php to "new course".

co author @g21emmka 

This will most likely solve https://github.com/HGustavs/LenaSYS/issues/13424

**Comment by c21sebar** [comment]
The bfs and fetchgithubrepo.php functions has been redesigned and the functions has been implemented under the refresh button on the navigation meny at the top.

The bfs function now look at an option parameter to be able to download and insert to sqlite or just insert to sqlite. We also changed so that when creating a new course all the files from the repo are not created on the webserver, the metadata from the files are added to the sqlite db. We tested that the files can be created with the DOWNLOAD parameter and it works.

The refresh button in the navbar is set to the REFRESH parameter to fetch the metadata from the githubrepo only if there has been a new commit to the repo.
